### DISCUSSION TITLE
About the formation and exportation of texture

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#1873] [Ask Anything] [answered]
Hi everyone
I've created a polycrystalline model in ABAQUS, and I want to use MTEX to input the orientation of grains.
I want to implement three things:
1. Input texture for a specific number of grains. For example 500 grains
2. Specific texture content. For example 0.3Brass+0.2Cube+0.5Radom; 0.2S+0.3Cube+0.5Brass
3. Export the information on the orientation in the type of Euler angles or rotation matrix
I'm hoping someone can tell me how to achieve it, thanks all.

**Comment by kilir** [comment, marked as answer]
Hi,
wrt1) I guess you need to be more specific, otherwise, i'd say load your 500 grains using the appropriate interface.
wrt2) have look at the documentation, simple define those odfs and add them using your weights which should add to 1.
wrt3) have a look at `orientation/export`
Cheers,
Rüdiger
